data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -282,55 +366,72 @@ def _pollution_factor(self) -> bool:
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.NPL_FIELD + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.TSDF_FIELD + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
+        pollution_eligibility_columns = [
+            field_names.RMP_LOW_INCOME_FIELD,
+            field_names.SUPERFUND_LOW_INCOME_FIELD,
+            field_names.HAZARDOUS_WASTE_LOW_INCOME_FIELD,
+        ]
+
+        rmp_sites_threshold = (
+            self.df[field_names.RMP_FIELD + field_names.PERCENTILE_FIELD_SUFFIX]
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
         )
 
-        return pollution_criteria & (
+        npl_sites_threshold = (
+            self.df[field_names.NPL_FIELD + field_names.PERCENTILE_FIELD_SUFFIX]
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
+        )
+
+        tsdf_sites_threshold = (
             self.df[
-                field_names.POVERTY_LESS_THAN_200_FPL_FIELD
-                + field_names.PERCENTILE_FIELD_SUFFIX
+                field_names.TSDF_FIELD + field_names.PERCENTILE_FIELD_SUFFIX
             ]
-            >= self.LOW_INCOME_THRESHOLD
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
+        )
+
+        # individual series-by-series
+        self.df[field_names.RMP_LOW_INCOME_FIELD] = (
+            rmp_sites_threshold & self.df[field_names.FPL_200_SERIES]
+        )
+        self.df[field_names.SUPERFUND_LOW_INCOME_FIELD] = (
+            npl_sites_threshold & self.df[field_names.FPL_200_SERIES]
+        )
+        self.df[field_names.HAZARDOUS_WASTE_LOW_INCOME_FIELD] = (
+            tsdf_sites_threshold & self.df[field_names.FPL_200_SERIES]
         )
 
+        self._increment_total_eligibility_exceeded(
+            pollution_eligibility_columns
+        )
+
+        return self.df[pollution_eligibility_columns].any(axis="columns")
+
     def _water_factor(self) -> bool:
         # In Xth percentile or above for wastewater discharge (Source: EPA Risk-Screening Environmental Indicators (RSEI) Model)
         # AND
         # Low income: In 60th percentile or above for percent of block group population
         # of households where household income is less than or equal to twice the federal
         # poverty level. Source: Census's American Community Survey]
-        return (
-            self.df[
-                field_names.POVERTY_LESS_THAN_200_FPL_FIELD
-                + field_names.PERCENTILE_FIELD_SUFFIX
-            ]
-            >= self.LOW_INCOME_THRESHOLD
-        ) & (
+
+        wastewater_threshold = (
             self.df[
-                field_names.WASTEWATER_FIELD
+                field_names.WASTEWATER_DISCHARGE_LOW_INCOME
```

Comment:
this should be `field_names.WASTEWATER_FIELD` not `field_names.WASTEWATER_DISCHARGE_LOW_INCOME`